app/cdap/components/Connections/Create/ConnectionConfiguration/ConnectionConfigForm.tsx

```diff
@@ -27,17 +29,30 @@ const useStyle = makeStyle(() => {
       gap: '10px',
       paddingLeft: '10px',
     },
+    connectionTestMessage: {
+      display: 'flex',
+      height: '2rem',
+      padding: '0 10px',
+    },
+    connectionTestSuccess: {
+      color: '#4ab63c',
+    },
+    connectionTestFailure: {
+      color: '#d15668',
```

Comment:
are these not theme colors?

Reply:
Looks like the failure color is `red-03`. The green color is not in the theme but it looks like they're not in sorted order so I'll add it.